src/Tracking.php

```diff
@@ -81,6 +81,11 @@ public function __construct( array $trackers = array() ) {
 	 * @return void
 	 */
 	public function handle_page_visit() {
+		if ( ! ( is_page() || is_product() ) ) {
```

Comment:
This means that we are tracking only in page and product, which means we won't be tracking on blog posts, i.e. we won't know conversion from blog posts (i.e. if users landed on a blog post / news post / promotion post, and then proceed to buy stuff)?